packages/inline-definition/src/InlineDefinition.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip, { TooltipProps } from '@leafygreen-ui/tooltip';
+import { Body } from '@leafygreen-ui/typography';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const underline = css`
+  display: inline-flex;
+  background-repeat: repeat-x;
+  background-position: center bottom;
+  background-size: 3px 2px;
+  transition: background-image 150ms ease-in-out;
+
+  background-image: radial-gradient(
+    circle closest-side,
+    ${uiColors.gray.dark1} 75%,
+    transparent 25%
+  );
+
+  &:hover {
+    background-image: radial-gradient(
+      circle closest-side,
+      currentColor 75%,
+      transparent 25%
+    );
+  }
+`;
+
+const maxWidth = css`
+  max-width: 240px;
+`;
+
+type InlineDefinitionProps = Partial<TooltipProps> & {
+  definition: string;
+};
+
+function InlineDefinition({
```

Comment:
I'm not sure why, but it appears that there's a div that's inserted in the trigger span element when hovering on the inline definition element. That seems to be causing some DOM nesting validation errors.

Additionally, the Tooltip overall isn't consistently being displayed on hover – its behavior is quite erratic when interacting with the element, and it's difficult to persist the tooltip on the page when hovering.

Reply:
I'm actually seeing this erratic behavior on the Tooltip component on master: https://mongodb.github.io/leafygreen-ui/?path=/story/tooltip--uncontrolled

Reply:
Seems like something is wrong with popover: https://mongodb.github.io/leafygreen-ui/?path=/story/popover--default

Reply:
It's happening specifically with `usePortal={true}` so I wonder if it's even coming from updates to the Portal component... checking git history to see what's up

Reply:
Confirming the bug was introduced in v5.0.0 of Popover. Hard setting the tooltip dep to v3.2.0 works, but v3.2.1 does not